### DISCUSSION TITLE
Boat routing does not work in Stockholm area

### DISCUSSION THREAD

**Discussion by r1ckdu** [osmandapp/OsmAnd#18452] [Maps Related]
Hi there, 

I have downloaded in osmand+ the Stockholm map and tried using boat navigation. The navigation returns a route that exits the maps border in a straight line and returns from that point again. Using the same settings on a map of greater london follows the water ways as expected. Similar results are obtained for Mecklenburg Vorpommern around the Waren area. 

So what's wrong with Sweden? How to fix? I'd like to easily judge how far my route will be without manually putting points on the map ^^

Thanks for your thoughts!

**Comment by tommycrock** [comment]
Screenshots of the situation in Stockholm and somewhere successful will probably help others.

**Comment by DmitryAlexei** [comment]
According to documentation, due to lack of most Fairways in OpenStreeetMap data, the boat routing is not recommended for open water navigation, it is suggested to use [Direct to point](https://osmand.net/docs/user/navigation/routing/boat-navigation#direct-to-point-routing-for-boat) instead.
See [Boat routing navigation article](https://osmand.net/docs/user/navigation/routing/boat-navigation).
